backend/oasst_backend/models/user.py

```diff
@@ -23,6 +23,10 @@ class User(SQLModel, table=True):
     created_date: Optional[datetime] = Field(
         sa_column=sa.Column(sa.DateTime(), nullable=False, server_default=sa.func.current_timestamp())
     )
+    stats_enabled: bool = Field(
+        nullable=False, sa_column=sa.Column(sa.Boolean, nullable=False, server_default=sa.true())
```

Comment:
probably the `nullable` is ignored when `sa_column` is specified, but no problem.